src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -36,6 +36,46 @@
 */
 public with sharing class BGE_ConfigurationWizard_CTRL {
 
+    /*******************************************************************************************************
+    * @description These fields are required only if at least 1 field from its object group is selected
+    */
+    private static Set<String> systemRequiredFields = new Set<String>{
```

Comment:
This is great to be up here for easy access! Maybe we could also put our description field up here? 🤞

Reply:
....but sadly not quite as simple because it's dependent on sObjectName.